salt/utils/platform.py

```diff
@@ -177,3 +178,13 @@ def is_fedora():
         x.strip('"').strip("'") for x in linux_distribution()
     ]
     return osname == "Fedora"
+
+
+@real_memoize
+def spanwing_platform():
+    """
+    Returns True if multiprocessing.get_start_method(allow_none=False) returns "spawn"
+
+    This is the default for Windows and macOS on Python >= 3.4
```

Comment:
Per the python docs [here](https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods). The default start method on macOS isn't changed to `spawn` until `3.8`

Reply:
Fixed. Thanks!